File: program/program/print-all-prime-numbers-between-two-numbers/print_all_prime_numbers_between_two_numbers.dart
void primeNos(int start , int end){
  for(var i = start ; i <= end ; i++){
    int factors = 0;
    for(var j = 1 ; j <= i ; j++){
      if(i % j == 0){
        factors += 1;
      }
    }
    if(factors <= 2){
        print(i);
    }
  }
}

void main() {
  primeNos(10 , 20);
}

File: program/program/print-all-prime-numbers-between-two-numbers/PrintAllPrimeNumbersBetweenTwoNumbers.cpp
#include<bits/stdc++.h>
using namespace std;
void SioErato(int n)
{
    int i;
    bool t[n + 1];
    fill(t, t + n + 1, true);
    t[0] = false, t[1] = false;
    for (i = 2; i * i <= n; i++)
    {
        for (int j = 2 * i; j <= n; j += i)
            if (j % i == 0)
                t[j] = false;
    }
    for (i = 1; i <= n; i++)
        if (t[i] == true)
            cout << i << " ";
}
int main()
{
    int a,b;
    cout<<"Enter range start and range end: ";
    cin>>a>>b;
    for (int i = a; i <= b; i++){
        if (i == 1 || i == 0)
            continue;
        int f = 1;
        for (int j = 2; j <= i / 2; ++j)
            if (i % j == 0){
                f = 0;
                break;
            }
        if (f == 1)
            cout << i << " ";
    }
    return 0;
}